tests/gdsconv/black_box_test/hsc_black_box_test/high_speed_comparator.const.json

```diff
@@ -1,33 +1,140 @@
 [
-    {"constraint": "PowerPorts", "ports": ["VCC"]},
-    {"constraint": "GroundPorts", "ports": ["VSS"]},
-    {"constraint": "ClockPorts", "ports": ["clk"]},
-    {"constraint":"HorizontalDistance", "abs_distance":0},
-    {"constraint":"VerticalDistance",   "abs_distance":0},
-    {"constraint":"GroupBlocks",  "instances": ["mn3", "mn4"],   "name": "ccn"},
-    {"constraint": "GroupBlocks", "instances": ["mp5", "mp6"],   "name": "ccp"},
-    {"constraint": "SymmetricBlocks",
-        "direction" : "V",
-        "pairs": [["mn0"], ["dp0"], ["ccn"], ["ccp"], ["mp7", "mp8"], ["mp9", "mp10"], ["xi0", "xi1"]]
-    },
-    {"constraint": "Order",
-        "direction" : "top_to_bottom",
-        "instances": ["mn0", "dp0", "ccn", "ccp"]
-    },
-    {"constraint": "Align",
-        "line" : "h_bottom",
-        "instances": ["mp9", "mp7", "dp0", "mp8", "mp10"]
-    },
-    {"constraint": "Align",
-        "line" : "h_bottom",
-        "instances": ["xi0", "ccp", "xi1"]
+    {
```

Comment:
What is the reason for changing this file?

Reply:
changed it to keep it same as the one in examples/ directory.